tests/ptf_runner.py

```diff
@@ -5,6 +5,11 @@
 logger = logging.getLogger(__name__)
 
 
+def ptf_collect(host, log_file):
+    host.fetch(src=log_file, dest='./logs/ptf_collect/', flat=True, fail_on_missing=False)
+    pcap_file = log_file[0:-3] + 'pcap'
```

Comment:
This logic should be enhanced, what if the log-file is not ended as ".log"?  
like: "lg", then it may raise exception since the log file name length is less than 3.

Reply:
1. lg
2. lg.abcd
3. logger

Reply:
Yeah, that make sense.